docs/vc-http-api.yml

```diff
@@ -41,6 +41,28 @@ paths:
           description: invalid input!
         "500":
           description: error!
+  /credentials/updateStatus:
+    post:
+      tags:
+        - Issuer
+      summary: Updates the status of an issued credential
+      operationId: updateCredentialStatus
+      description: Updates the status of an issued credential.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/UpdateCredentialStatus"
+        description: Parameters for updating the status of the issued credential.
+      responses:
+        "200":
+          description: Credential status successfully updated!
+        "400":
+          description: invalid input!
+        "404":
+          description: credential not found!
+        "500":
+          description: error!
```

Comment:
```suggestion
          description: Internal server error.
```

Reply:
We really need to describe when these sorts of errors can occur.

Reply:
What error is thrown if the status list type is not supported?

Reply:
Agree we need more details on the error cases, will add some now, I would assume it would be a 400 because you are making a request that the server does not understand.